contrib/debian/examples/bitcoin.conf

```diff
@@ -131,6 +131,10 @@
 # be validated sooner.
 #paytxfee=0.00
 
+# Prune your node to the last n blocks. 550 is the minimum. 
+# Pruning can drastically reduce the amount of disk space required to run bitcoin down to ~3GB.
```

Comment:
You have the unit wrong, it's not in blocks but in MiB. Maybe you can borrow some description text from the help message:
```
  -prune=<n>
       Reduce storage requirements by enabling pruning (deleting) of old
       blocks. This allows the pruneblockchain RPC to be called to
       delete specific blocks, and enables automatic pruning of old
       blocks if a target size in MiB is provided. This mode is
       incompatible with -txindex and -rescan. Warning: Reverting this
       setting requires re-downloading the entire blockchain. (default:
       0 = disable pruning blocks, 1 = allow manual pruning via RPC,
       >550 = automatically prune block files to stay under the
       specified target size in MiB)
```